### DISCUSSION TITLE
Github Student Request Approved on 21st Sept but still havent received the benefits

### DISCUSSION THREAD

**Discussion by ruturaj-ghatage** [community/community#139924] [GitHub Education]
### Hi everyone,

Github Student Request Approved on 21st Sept but still havent received the benefits

**Comment by Wolfieleo** [comment]
mine was approved on the same date, still no benefits

**Comment by Jack026** [comment]
Same here

**Comment by davevad93** [comment]
Hi @ruturaj-ghatage ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @ruturaj-ghatage was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
